aiogram/types/base.py

```diff
@@ -259,7 +260,7 @@ def iter_values(self):
         for _, value in self:
             yield value
 
-    def __hash__(self):
+    def __hash__(self) -> str:
```

Comment:
Hash must always return an integer (read the python docs)

```suggestion
    def __hash__(self) -> int:
```